docs/reference/filters.md

```diff
@@ -944,6 +944,26 @@ Example:
 auditLog()
 ```
 
+## unverifiedAuditLog
+
+Filter `unverifiedAuditLog()` adds a Header, `X-Unverified-Audit`, to the request, the content of which, will also
+be written to the log file. By default, the value of the audit header will be equal to the value of the `sub` key, from
+the Authorization token. This can be changed by providing a `string` input to the filter which matches another key from the
+token.
+
+*N.B.* It is important to note that, if the content of the `X-Unverified-Audit` header does not match the following regex, then
+a default value of `invalid-sub` will be populated in the header instead:
+    `^[a-zA-z0-9_/:?=&%@.#-]*$`    
```

Comment:
What's the reasoning behind this restriction?

Reply:
This was an existing restriction that was in place. Maybe @szuecs can shine some light.

Reply:
The commit message was:
```
commit 81013aefcfa07d4fac9b3c8f75d4752dbcb1f0a4 (tag: v0.10.2)
Author: Sandor Szücs <sandor.szuecs@zalando.de>
Date:   Thu May 3 12:01:01 2018 +0200

    fix prepared wrong garbage in unverified auditlog (#639)

    * fix prepared wrong garbage in unveirfied auditlog

    * add URI Sub test and change the defaultSub if we do not match Sub data
```